scripts/label_all_tokens.py

```diff
@@ -86,8 +91,13 @@ def main():
         labelled_token_ids_dict[token_id] = labels[0][0]
 
     # Save the labelled tokens to a file
-    filename = "labelled_token_ids_dict.pkl"
-    filepath = STATIC_ASSETS_DIR.joinpath(filename)
+    if os.path.split(args.output)[0] == "":
+        filepath = STATIC_ASSETS_DIR.joinpath(args.output)
+        print(f"Outputting file {args.output} to path {filepath}")
+    else:
+        filepath = os.path.expandvars(args.output)
+        print(f"Outputting to path {filepath}")
+
     with open(f"{filepath}", "wb") as f:
```

Comment:
```suggestion
    with open(filepath, "wb") as f:
```

Reply:
`open()` can't be used on the `Traversable` type that `filepath` is. The `Traversable.open()` class method also cannot be used, as it only works for reading mode. I leave this as `open(str(filepath))` for now